formV.js

```diff
@@ -0,0 +1,19 @@
+const form = document.querySelector('#form');
+const email = document.querySelector('#email');
+const errorMsg = document.querySelector('#msg-error');
+const ERROR_MESSAGE = 'Please use lowercase to share your email address.';
+
+function isLowerCase(str) {
+  return str === str.toLowerCase();
+}
+
+form.addEventListener('submit', (event) => {
+  if (!isLowerCase(email.value)) {
+    errorMsg.innerText = ERROR_MESSAGE;
+    email.classList.add('email-input-error');
+    event.preventDefault();
+  } else {
+    errorMsg.innerText = '';
+    email.classList.add('email-input-error');
+  }
+});
```

Comment:
## [OPTINAL]

- [ ]  To improve the design I would suggest you add some styling error interaction on the input that contains the validation error. Just consider it as a challenge that can improve your UI.  @sergiogval   [see the challenge here in its form](https://andersonlebon.github.io/Portfolio-w2/). 
![image](https://user-images.githubusercontent.com/65068771/146368315-34dae144-e2d0-4e7a-a7b4-6061e59892b0.png)
![image](https://user-images.githubusercontent.com/65068771/146368744-375816e7-f4d1-42d1-875b-ae22fa593453.png)